### PR TITLE
Add tolerations and affinity to harbor

### PR DESCRIPTION
**Pull request by ITViking**
#### What does this PR do?
It should make Kubernetes schedule Harbor on our system nodes 

#### Should this be tested by the reviewer and how?
No

#### Any specific requests for how the PR should be reviewed?
Read it throug

#### What are the relevant tickets?
https://reload.atlassian.net/jira/software/c/projects/DDFDRIFT/boards/464?selectedIssue=DDFDRIFT-86

### CODE REVIEW

File: infrastructure/environments/dplplat01/configuration/harbor/harbor-values.template.yaml [hunk 1]
```diff
@@ -61,3 +61,20 @@ persistence:
       accountname: "$STORAGE_ACCOUNT_NAME"
       accountkey: "$STORAGE_ACCOUNT_KEY"
       container: "$STORAGE_CONTAINER_NAME"
+
+# Use tolerations
+tolerations:
+  - key: noderole.dplplatform
+    effect: NoSchedule
+    operator: Equal
+    value: system
```
Comment (ath88) [inline infrastructure/environments/dplplat01/configuration/harbor/harbor-values.template.yaml:70]: Its the admin node - also you need to fix the effect.

**Comment by ath88** [comment]
Harbor has several components - are you able to set a general toleration and affinity like this? Have you looked into the helm chart docs?

File: infrastructure/environments/dplplat01/configuration/harbor/harbor-values.template.yaml (hunk 1 above)
Comment (ITViking) [reply infrastructure/environments/dplplat01/configuration/harbor/harbor-values.template.yaml:70]: oh right

**Comment by ITViking** [comment]
Good point - I'll look into that

**Comment by ITViking** [comment]
fixed it

**Review by ath88 — APPROVED** [review]
merge at will

**Pull request merged**